lib/active_model/serializer/adapter/nested_json.rb

```diff
@@ -0,0 +1,57 @@
+module ActiveModel
+  class Serializer
+    class Adapter
+      class NestedJson < Json
+        cattr_accessor :default_limit_depth
+        @@default_limit_depth = 5
+
+        def serializable_hash options = {}
```

Comment:
That is for `#fragment_cache`.

The way we cache fragmentation is exactly the same to `Adapter::Json` ( like `Adapter::FlattenJson` as well).
And `Adapter::JsonApi` caches differently.